### ISSUE TITLE
Compliance article 5

### ISSUE THREAD

**Issue by Rakedjian** [OIEau/uwwtd#366]
There is a problem with the compliance calculation under Article 5. 

How can we consider that this UWWTP is compliant 
http://webnuxdev.rnde.tm.fr/uwwtd_ro/treatment-plant/ro1nt_53/2014

if in the agglomeration calculation the distance to compliance is not equal to zero

http://webnuxdev.rnde.tm.fr/uwwtd_ro/agglomeration/roag_120735/2014

We have also in the register an issue with the article 5 that seems to be ok but when we compare the information related to the treatment in place and the treatment require it is not ok

![pietra neam ro](https://cloud.githubusercontent.com/assets/16286544/21226700/f2781d14-c2d6-11e6-9020-fac60dd917d7.GIF)

![peitra neam treatment](https://cloud.githubusercontent.com/assets/16286544/21226705/f6bc016a-c2d6-11e6-938a-91d3909122bb.GIF)

**Comment by friboben** [comment]
a) The case raised is complex: Pending deadline for art4 and 5 but deadline passed for art 3. as when considering art3 it is NC, the agglo is NC even if the TP is C (discharge without treatment, hence we can not say it will go in TP or not, hence TP is compliant).
The only issue I see with the agglo sheet is that compliance art 4 and art5 should be NR because it is pending, but as deadline is passed for art3, overall compliance for agglo is to be NC.
b) has to be checked

**Comment by Rakedjian** [comment]
We only use the load for the compliance calculation I will send you my assessment

**Comment by cnassar** [comment]
Fixed partially. Now, if article 4 or 5 are under PD and article 3 is NC. Article 4 and 5 will be shown as not relevant.
It still the problem of the table distance to compliance on the website. The performance should be applied only for Premoval and Nremoval, and not for other treatment.
However I did not understand the problem with the register. The fact is that only one column is provided to know if we are under pending or expired deadline (for the 3 articles) : it is column G in the sheet agglomeration level.  As the deadlines are different, this column is relevant only for article 3.
It need to be developped as an inhancement.
